src/lib.rs

```diff
@@ -20,12 +20,19 @@
 //! assert_eq!(17, seventeen.into_inner());
 //! assert_eq!("XVII", seventeen.to_string());
 //! ```
+#![deny(
+    unsafe_code,
+    missing_docs,
```

Comment:
"Deny missing docs" is a bad idea in any code base. The more code you have, the worse this gets. Look in any modern line of business application where this is in use and you will absolutely find code like...
```csharp
/// <summary>Gets the user by id</summary>
public User GetUserById(int id)
```